README.md

```diff
@@ -1,202 +1,81 @@
-# 🏃🏽‍♀️🏃🏾‍♂️🏃🏻‍♀️💨 _⸻RUN_
+# 🏃🏽‍♀️🏃🏾‍♂️🏃🏻‍♀️💨 **_⸻ RUN_**
```

Comment:
reviewer note:
even though github says this adds 375 lines of README.md and removes 302 lines of NEW_README.md, github is lying: it really is just an `mv`.

```
git diff eric/ecs/readme-refresh^:NEW_README.md eric/ecs/readme-refresh:README.md
```